### DISCUSSION TITLE
Negative Test with wrong receiver port 4317

### DISCUSSION THREAD

**Discussion by alokmajumder** [open-telemetry/opentelemetry-java#2909] [Q&A]
Hi, I was trying to do a negative testing to send span and traces to wrong port 4310. Default otel port is 4317. But still it seems my code is sending to 4317. My expectation was it will fail to send on 4317.
My command to start the application is
java -Dotel.exporter.otlp.endpoint=localhost:4310 -jar target/example-jaeger-jar-with-dependencies.jar localhost 14250
github location for code is https://github.com/alokmajumder/openTelemetry-Poc.git (edited)

**Comment by jkwatson** [comment]
It appears that your example is manually configuring the Jaeger exporter and not using any auto-configured OTLP exporter. The environment variables/system properties are only supported via the autoconfiguration system, not in general.

Here is your code where you configure the Jaeger exporter:  https://github.com/alokmajumder/openTelemetry-Poc/blob/main/src/main/java/io/opentelemetry/example/jaeger/JaegerConfiguration.java#L25-L29

If you want to use autoconfiguration, you'll need to configure your sdk using the autoconfiguration mechanisms: https://github.com/open-telemetry/opentelemetry-java/blob/main/QUICKSTART.md#auto-configuration
